lib/geomancer/geo_json/feature_set.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Geomancer.GeoJson.FeatureSet do
```

Comment:
With all of these maps and zips, this module displays an intimate knowledge of the structures of GeoJson and Shape files. I think a couple of 'why' comments would be super helpful to those who are reading this code in the future without as intimate a knowledge.

Reply:
Yeah, definitely. Comments, moduledocs, and function docs are incoming.